core/src/main/java/com/linecorp/armeria/client/ClientFactory.java

```diff
@@ -128,6 +128,11 @@ static void disableShutdownHook() {
      */
     MeterRegistry meterRegistry();
 
+    /**
+     * Returns the {@link ClientFactoryOptions} that has been used to create this {@link ClientFactory}.
+     */
+    ClientFactoryOptions options();
```

Comment:
Could you move this method after `setMeterRegistry()`, so that `meterRegistry()` and `setMeterRegistry()` are grouped together?